appveyor.yml

```diff
@@ -173,7 +170,21 @@ for:
   deploy_script: |
     set -eo pipefail
     if [[ $APPVEYOR_REPO_TAG == "true" && $TOXENV == $DEPLOY_TOXENV ]]; then
-      pip install twine
-      python setup.py sdist
-      twine upload --skip-existing dist/*.gz
+      pip install twine wheel pdm
+      # source distribution
+      pdm build --no-wheel
+
+      # wheels
+      mv x86_64/LICENSE docs
+      # wheel x86_64
+      mv x86_64/lib/libmediainfo.so.0.0.0 pymediainfo/libmediainfo.so.0
+      pdm build --no-sdist
+      wheel tags --remove --platform-tag=manylinux_2_34-x86_64 dist/*-py3-none-any.whl
+      # wheel arm64
+      mv -f arm64/lib/libmediainfo.so.0.0.0 pymediainfo/libmediainfo.so.0
```

Comment:
In the code, the default is to use `library_paths = ("libmediainfo.so.0",)`. This means that this library expects it to exist. If it doesn't, there's something wrong and I'll have to update the code, not only the CI.
However, nothing guarantees that the real file is named `libmediainfo.so.0.0.0`. That's why I thought it would be better to copy `libmediainfo.so.0` (cp resolves the link to copy the actual file).
Please let me know if this is clear.

Reply:
OK, but it's renamed to .so.0 (`mv` moves and renames files). It's a common practice to have the real lib file with full version and symlink to .so.0 and .so. Therefore I doubt mediainfo will change the naming scheme, but as you prefer.

Reply:
Oh I know your command renames the file to what is expected.
All I'm saying is that there is no guarantee that upstream won't change the filename later from `libmediainfo.so.0.0.0` to something. On the other hand, since `libmediainfo.so.0` is the `SONAME`, I think it is less likely for them to rename `libmediainfo.so.0` without a major reason.
```
objdump -p libmediainfo.so.0 | grep SONAME
  SONAME               libmediainfo.so.0
```
@jeromemartinez please let me know if my idea of copying the `.so.0.0` by resolving the symlink from `.so.0` makes more sense than directly copying the `.so.0.0`. Also, do you have an opinion on us bundling the AWS Lambda library into the Python wheels?